feature 1443 document probabilistic field settings (#1508)
dtcenter/METplus: docs/Users_Guide/systemconfiguration.rst

it is not set, the values in the corresponding
FCST_VAR<n>_[NAME/LEVELS/THRESH/OPTIONS] will be used in the ens dictionary.

Probabilistic Forecast Fields
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

If processing probabilistic forecast fields, there are additional configuration
variables that are used to properly format the field info that is passed into
the wrapped MET configuration files.
:term:`FCST_IS_PROB` is used to process probabilistic data::

    [config]
    FCST_IS_PROB = True

>     FCST_VAR1_NAME = APCP

Reviewer: Wondering if we should use a more likely example, like verifying the ensemble frequencies from the NetCDF output of Ensemble-Stat or Gen-Ens-Prod?
```suggestion
    FCST_VAR1_NAME = APCP_24_A24_ENS_FREQ_gt0.0
```

    FCST_VAR1_LEVELS = "(*,*)"

will add the following to the MET config file::

>     fcst = {field = [{ name="APCP"; level="(*,*)"; prob=TRUE; cat_thresh=[ ==0.1 ]; }];}

Reviewer: If you switch, you'd need to update this line too:
```suggestion
    fcst = {field = [{ name="APCP_24_A24_ENS_FREQ_gt0.0"; level="(*,*)"; prob=TRUE; cat_thresh=[ ==0.1 ]; }];}
```

> The cat_thresh value used in this case defaults to ==0.1.

Reviewer: ```suggestion
The cat_thresh value defaults to ==0.1 and defines the size of the Nx2 probabilistic contingency table.
```
